controllers/enforcer.go

```diff
@@ -135,15 +135,27 @@ func (c *ApiController) BatchEnforce() {
 	}
 
 	res := [][]bool{}
+	listPermissionIdMap := map[string][]string{}
+
 	for _, permission := range permissions {
-		enforceResult, err := object.BatchEnforce(permission.GetId(), &requests)
+		permissionIds, ok := listPermissionIdMap[permission.Adapter]
+		if !ok {
+			listPermissionIdMap[permission.Adapter] = []string{permission.GetId()}
```

Comment:
Why only use adapter as map key? Why not include model too? If model is different, will the enforcer work the same?

Reply:
Because we get these permissions by the modelId, so it's surely the same model

Reply:
But `modelId` is only one of all the ways, it's optional. You need to make sure other ways work too

Reply:
@hsluoyz okay, fixed it